src/views/ManageList.jsx

```diff
@@ -1,7 +1,74 @@
+import { addItem } from '../api/firebase';
+import toast, { Toaster } from 'react-hot-toast';
+
 export function ManageList() {
+	const notify = () => toast.success('Item added to list!');
+
+	const handleSubmit = (e) => {
+		e.preventDefault();
+		const itemInfoForm = new FormData(e.target);
+
+		console.log({
+			itemName: itemInfoForm.get('item'),
+			daysUntilNextPurchaseSelect: itemInfoForm.get('when-to-buy-select'),
```

Comment:
When we pick which select we like we only need to keep one of them, but wanted to make sure the infor actually passed.

Reply:
I think I figured out my question about the listPath value (yes middle of the night after a migraine 😅) in `app.jsx` for the call that has the `/manage-list` we can add the list path value there to add it. I believe it is in a different route there already. (I'm on my phone so hard to check)